modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/sql/IgniteSqlKillWaitMode.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.sql.engine.sql;
+
+/** Wait mode used by a KILL statement. */
+public enum IgniteSqlKillWaitMode {
```

Comment:
nitpicking: perhaps, it would be better to replace this enum with simple boolean flag. First, options other of the enumeration is kinda strange: we have only `implicit` and `no_wait`. But what if implicit became `no_wait` as well? It's better to rename to `wait` to make it obvious. No, we have enum that's clearly maps to a boolean flag

Reply:
Fixed. Converted to boolean.